www/front_src/src/Resources/Graph/Performance/Graph/index.tsx

```diff
@@ -482,4 +487,58 @@ const Graph = ({
   );
 };
 
+const memoProps = [
+  'addCommentTooltipLeft',
+  'addCommentTooltipTop',
+  'addCommentTooltipOpen',
+  'width',
+  'height',
+  'timeSeries',
+  'base',
+  'lines',
+  'xAxisTickFormat',
+  'timeline',
+  'tooltipPosition',
+  'resource',
+  'eventAnnotationsActive',
+];
+
+const MemoizedGraphContent = memoizeComponent<GraphContentProps>({
+  memoProps,
+  Component: GraphContent,
+});
+
+const Graph = (
+  props: Omit<
+    GraphContentProps,
+    | 'addCommentTooltipLeft'
+    | 'addCommentTooltipTop'
+    | 'addCommentTooltipOpen'
+    | 'showAddCommentTooltip'
+    | 'hideAddCommentTooltip'
+    | 'format'
+  >,
+): JSX.Element => {
+  const { format } = useLocaleDateTimeFormat();
+  const {
+    tooltipLeft: addCommentTooltipLeft,
+    tooltipTop: addCommentTooltipTop,
+    tooltipOpen: addCommentTooltipOpen,
+    showTooltip: showAddCommentTooltip,
+    hideTooltip: hideAddCommentTooltip,
+  } = useTooltip();
```

Comment:
Why do we initialize the comment tooltip here whereas the metrics tooltip is being initialized within the "MemoizedGraphContent" ?

Reply:
Because `showAddCommentTooltip` and `hideAddCommentTooltip` are functions which cause a re-render